front/components/vaults/VaultCreateWebsiteModal.tsx

```diff
@@ -39,24 +39,27 @@ import { SendNotificationsContext } from "@app/components/sparkle/Notification";
 import { useVaultDataSourceViews } from "@app/lib/swr";
 import { isUrlValid, urlToDataSourceName } from "@app/lib/webcrawler";
 import type { PostManagedDataSourceRequestBodySchema } from "@app/pages/api/w/[wId]/data_sources/managed";
+import type { PostVaultDataSourceResponseBody } from "@app/pages/api/w/[wId]/vaults/[vId]/data_sources/static";
 
 const WEBSITE_CAT: DataSourceViewCategory = "website";
 
+// todo(GROUPS_INFRA): current component has been mostly copy pasted from the WebsiteConfiguration existing component
+// this should be refactored to use the new design.
 export default function VaultCreateWebsiteModal({
   isOpen,
   setOpen,
   owner,
   dataSources,
   vault,
-  dataSourceOrView,
+  dataSourceView,
```

Comment:
Not sure to understand why we still need this one where as there is only one call site that always set it to `null` 🤔